include/seqan3/range/container/small_vector.hpp

```diff
@@ -949,7 +1079,10 @@ class small_vector
  * \{
  */
 
-//!\brief Deducts the size and value type from an built-in array on construction.
+/*!\brief Deducts the size and value type from an built-in array on construction.
+ * \details
+ * \experimentalapi{Experimental since version 3.1.}
+ */
 template <size_t capacity2, typename value_type>
 small_vector(const value_type (&array)[capacity2]) -> small_vector<value_type, capacity2>;
```

Comment:
Not sure if this is valid: (const position)
```suggestion
small_vector(value_type const (&array)[capacity2]) -> small_vector<value_type, capacity2>;
```

Reply:
works too xD